e2e/sbin/test.sh

```diff
@@ -26,7 +26,13 @@ $SBIN_DIR/wait-for-it.sh -t 60 $HOST:$L2_HTTP_PORT
 # Run testing script
 cd $CONTRACTS_DIR
 npx ts-node scripts/testing.ts
-RESULT=$?
+exit_0 = $?
```

Comment:
we probably want to separate these tests so we know which succeeded and which failed. we can put them into different github action targets.